bigvince/storage_backends.py

```diff
@@ -60,6 +61,7 @@ class SharedMediaStorage(S3Boto3Storage):
     #region_name = "us-east-1"
     acl = 'private'
     custom_domain = False
+    bucket_name = getattr(settings, 'VINCE_SHARED_BUCKET')
 
```

Comment:
With Django 3.2 all the __init__ overrides in this file should be gone.  It is a bit more simplified

Reply:
Will fix and re submit by tomorrow, thanks

Reply:
@sei-vsarvepalli 
Hi Vijay,
About to resubmit this. 

However, since we are now on Django 3+, I am having issues with %load staticfiles% found in templates/*.html 

ref:
https://docs.djangoproject.com/en/dev/releases/3.0/#features-removed-in-3-0

Replacing with %load static% seems to fix the issue. Curious that this doesn't affect the front end for your tests of main?

<img width="765" alt="Screenshot4" src="https://user-images.githubusercontent.com/19195413/218580482-e85cf0cb-1b9c-4811-9bd5-520948551dcd.png">

Thanks

Reply:
That should be a simple fix.. Try adding 

```
            'libraries': {
                'staticfiles': 'django.templatetags.static',
            }

```

to TEMPLATES in settings.py - will have to add this to settings_.py too for ease of use.